vms/platformvm/vm_regression_test.go

```diff
@@ -2235,7 +2235,7 @@ func buildAndAcceptStandardBlock(vm *VM) error {
 
 func checkValidatorBlsKeyIsSet(
 	valState validators.State,
-	nodeID ids.NodeID,
+	nodeID ids.ShortNodeID,
```

Comment:
This function ends up converting this into a generic nodeID - so seems like we should just take one in. 
```suggestion
	nodeID ids.NodeID,
```